packages/backend/src/ee/services/RolesService.ts

```diff
@@ -410,122 +365,47 @@ export class RolesService extends BaseService {
         return assignments;
     }
 
-    async createProjectRoleAssignment(
+    async updateProjectRoleAssignment(
```

Comment:
Is the plan to move these into OSS in another PR?

Reply:
I've already migrated roleService into OSS . But controllers are still in EE